.travis.yml

```diff
@@ -1,6 +1,7 @@
 language: ruby
 rvm:
   - 2.3.1
+  - 2.4.0
```

Comment:
The latest version of MRI is 2.4.1.
https://www.ruby-lang.org/en/news/2017/03/22/ruby-2-4-1-released/